shared/message/purpose/Defender.proto

```diff
@@ -23,3 +23,6 @@ package message.purpose;
 message Defender {
     bool force_playing = 1;
 }
+
+message NormalDefender {}
+message PenaltyShootoutDefender {}
```

Comment:
There technically isn't a defender role for penalty shootout, so we should just have the Defender message and no NormalDefender and PenaltyShootoutDefender

Reply:
Removed